easybuild/easyconfigs/b/BioPerl/BioPerl-1.7.2-foss-2019a-Perl-5.28.1.eb

```diff
@@ -0,0 +1,30 @@
+# easybuild easyconfig
+#
+#  John Dey jfdey@fredhutch.org
+#
+#  Fred Hutchinson Cancer Research Center
+
+easyblock = 'PerlModule'
+
+name = 'BioPerl'
+version = '1.7.2'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://www.bioperl.org/'
+description = """Bioperl is the product of a community effort to produce Perl code which is useful in biology.
+ Examples include Sequence objects, Alignment objects and database searching objects."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
Could this be moved up the toolchain hierarchy? I tried to skim to see if there was any potential LAPACK/BLAS dependencies here, and I don't think there is unless they hid it well.

Possibly not even any MPI stuff either? In fact, if it's all Perl scripts(?), then it seems to be it could possible to all the way down to GCCcore with no difference.

Reply:
Yep, currently other Perl packages are on `GCCcore`, so moved BioPerl to `GCCcore` as well.